javascript/ql/lib/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -1682,9 +1682,6 @@ class RegExpCreationNode extends DataFlow::SourceNode {
     result = this.(RegExpLiteralNode).getFlags()
   }
 
-  /** Holds if the constructed predicate has the `g` flag. */
-  predicate isGlobal() { RegExp::isGlobal(this.getFlags()) }
-
```

Comment:
Don't outright delete predicates like this. Instead keep the implementation but add a `deprecated` annotation, and add an explanation into the docstring as to why a predicate is deprecated.  

Deprecated predicates gets deleted after a little over a year.